vscode/src/ruby.ts

```diff
@@ -223,12 +223,7 @@ export class Ruby implements RubyInterface {
       this.yjitEnabled && (major > 3 || (major === 3 && minor >= 2));
 
     // Starting with Ruby 3.3 the server enables YJIT itself
-    const useYjit =
-      vscode.workspace.getConfiguration("rubyLsp").get("yjit") &&
-      major === 3 &&
-      minor === 2;
-
-    if (this.supportsYjit && useYjit) {
+    if (this.supportsYjit && major === 3 && minor === 2) {
```

Comment:
This is strange. Am I reading this line correctly, that it only enables YJIT on Ruby 3.2 and not on any other version, like 3.3?

Reply:
Oh, I just saw the comment above this line. In that case, can't we just move the exact version check to the definition of `supportsYjit` and make it:
```ts
this.supportsYjit = this.yjitEnabled && major == 3 && minor == 2;
```?

Reply:
And maybe rename that field to reflect the new meaning as well.